src/controllers/controller.bar.js

```diff
@@ -4,13 +4,14 @@ import {
   valueOrDefault, resolveObjectKey, sign, defined
 } from '../helpers';
 
-function getAllScaleValues(scale) {
+function getAllScaleValues(meta) {
+  let scale = meta.iScale;
   if (!scale._cache.$bar) {
-    const metas = scale.getMatchingVisibleMetas('bar');
+    const visiblemetas = scale.getMatchingVisibleMetas(meta._type);
```

Comment:
I would rename `visiblemetas` into `visibleMetas` to follow code writing convention and use camel case.

Reply:
Maybe you could change `meta._type` to `meta.type` here since that's how type is retrieve in the rest of source.

Reply:
rename visiblemetas to visibleMetas as you suggested.